Tasks/KubernetesV1/task.loc.json

```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 1,
     "Minor": 170,
-    "Patch": 1
+    "Patch": 2
```

Comment:
What am I missing? This version in Prod is `0.170.2` could you check if your earlier commit is present in `releases/m170` branch?

Reply:
Let me check.

Reply:
I checked in `release/170`, the version is as below. Earlier changes are not part of` m170`
```
    "version": {
        "Major": 1,
        "Minor": 169,
        "Patch": 2
    }
```